cartographer_ros/cartographer_ros/time_conversion.cc

```diff
@@ -17,31 +17,33 @@
 #include "cartographer_ros/time_conversion.h"
 
 #include "cartographer/common/time.h"
-#include "ros/ros.h"
+#include <rclcpp/rclcpp.hpp>
+#include <builtin_interfaces/msg/time.hpp>
```

Comment:
alphabetical order.
Consistency, let's pick `"` or `<` for all header from external packages and use the same everywhere